### ISSUE TITLE
Add BatchNormalization hyperparameter at the start

### ISSUE THREAD

**Issue by Marko-Shevchuk** [PNU-SE-AIteam/cifar-10-stable-diffusion-detection#64]

**Comment by sgorrister** [comment]
done
